src/components/Modules/Home/HomeIntroModule/HomeIntroModule.tsx

```diff
@@ -3,13 +3,15 @@ import "./custom.css";
 import { Inter } from "next/font/google";
 import Link from "next/link";
 import { LinkListComponent } from "~/components/Components/LinkListComponent/LinkListComponent";
+import AnimatedComponent from "~/components/Components/AnimatedComponent/AnimatedComponent";
 
 const inter = Inter({ subsets: ["latin"], variable: "--font-inter" });
 
 type NavLinks = {
```

Comment:
Do not re-type alias of Component/Atom 's type
juse use `React.ComponentProps<typeof {ComponentName}>;`